maps/nerva/nerva.dm

```diff
@@ -86,10 +86,10 @@
 	#include "../../code/datums/music_tracks/space_oddity.dm"*/
 
 	//keep your clothing hacks to yourself
-	#include "../torch/items/clothing/solgov-accessory.dm"
+	// #include "../torch/items/clothing/solgov-accessory.dm"	These don't seem to actually be used, but are causing issues with double-importing
```

Comment:
should we just move all the torch clothing the the dme? seems weird to include some of this stuff in noctis and some in nerva.

Reply:
Personally, I'd like to move towards a more modular approach, where each away/map/template only includes what it uses, and the .dme itself only includes the current map - dme loads nerva, nerva loads all required dependencies, or dme loads glloydstation, glloydstation loads its dependencies. Seems strange to me that we're loading so many things that aren't actually used.
Only issue with this approach is currently, although they claim otherwise, double-including seems to be an issue. Not sure if it's _actually_ being double-included, or if dreamchecker isn't smart enough to discard the extra include, but that causes errors.

Right now I'm playing around with some preprocessor macros to see if I can come up with some elegant solution to resolve double includes, but, these are a pain to debug, so I wouldn't count on it just yet.

Honestly happy to go with whatever solution you feel is best.

Reply:
My instinct is to keep things in the dme for use in events and such, but you're probably right that there's a lot that can be cut out. Either way, since nerva always loads noctis, it's probably fine for now.